docs/api/datasets/non_geo_datasets.csv

```diff
@@ -16,6 +16,7 @@ Dataset,Task,Source,License,# Samples,# Classes,Size (px),Resolution (m),Bands
 `Digital Typhoon`_,"C, R",Himawari,"CC-BY-4.0","189,364",8,512,5000,Infrared
 `ETCI2021 Flood Detection`_,S,Sentinel-1,-,"66,810",2,256x256,5--20,SAR
 `EuroSAT`_,C,Sentinel-2,"MIT","27,000",10,64x64,10,MSI
+`EverWatch`_,OD,Aerial,"CC0 1.0 Universal","5,325",7,"1,500x1500p",0.01,RGB
```

Comment:
```suggestion
`EverWatch`_,OD,Aerial,CC0-1.0,"5,325",7,"1,500x1500p",0.01,RGB
```
Names should match [SPDX identifiers](https://spdx.org/licenses/). Wish we had an easy way to test this, but there are also a lot of custom licenses.

Anyways, confirmed the license from https://zenodo.org/records/11165946